x-pack/legacy/plugins/apm/public/context/LicenseContext/index.tsx

```diff
@@ -3,33 +3,30 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+
 import React from 'react';
-import { FETCH_STATUS, useFetcher } from '../../hooks/useFetcher';
-import { loadLicense, LicenseApiResponse } from '../../services/rest/xpack';
```

Comment:
We can also remove the `loadLicense` method (and the entire `rest/xpack.ts` file)

Reply:
That file is deleted.